dev/io.openliberty.microprofile.metrics.internal.private.5.0/.classpath

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should the project name be `io.openliberty.microprofile.metrics.5.0.internal.private`?